### PR TITLE
Active Record and Active Job name formatting

### PR DESCRIPTION
**Pull request by andyatkinson**
Initial exploration of some formatting changes for consistency with Rails Guides and official sources:
https://guides.rubyonrails.org/index.html

- Active Job over ActiveJob
- Active Record over ActiveRecord

This requires close inspection though because ActiveJob and ActiveRecord can be valid parts of a class name or constant in the GoodJob source code.

https://github.com/bensheldon/good_job/issues/1048

### CODE REVIEW

**Comment by andyatkinson** [comment]
@bensheldon Low priority one, but let me know whether this is going in the correct direction. Getting early feedback here before spending too much time making changes that aren't correct or wanted.

File: app/models/good_job/execution.rb [hunk 1]
```diff
@@ -150,9 +150,9 @@ def self.queue_parser(string)
     # @return [ActiveRecord::Relation]
     scope :schedule_ordered, -> { order(coalesce_scheduled_at_created_at.asc) }
 
-    # Get Jobs were completed before the given timestamp. If no timestamp is
-    # provided, get all jobs that have been completed. By default, GoodJob
-    # destroys jobs after they are completed and this will find no jobs.
+    # Get completed jobs before the given timestamp. If no timestamp is
+    # provided, get *all* completed jobs. By default, GoodJob
+    # destroys jobs after they're completed, meaning this returns no jobs.
```
Comment (andyatkinson) [inline app/models/good_job/execution.rb:155]: Can back this out, but this was a little funky. Do you use capital-j Job a lot in comments to refer to Job instances? Could these usages be lowercase j generic "jobs"?

File: app/models/good_job/job.rb [hunk 2]
```diff
@@ -60,7 +60,7 @@ def table_name=(_value)
 
     scope :unfinished_undiscrete, -> { where(finished_at: nil, retried_good_job_id: nil, is_discrete: [nil, false]) }
 
-    # The job's ActiveJob UUID
+    # The job's Active Job UUID
```
Comment (andyatkinson) [inline app/models/good_job/job.rb:63]: This is a UUID that's turned into a string?

Comment (bensheldon) [reply app/models/good_job/job.rb:63]: Yep, that's how Ruby represents UUIDs:

```ruby
❯ bin/rails c
Loading development environment (Rails 7.1.1)
[1] pry(main)> SecureRandom.uuid.class
=> String
```

File: app/models/good_job/execution.rb (hunk 1 above)
Comment (bensheldon) [reply app/models/good_job/execution.rb:155]: this is ok 👍🏻 Reading through this, I think there's some historical baggage here given this is in `GoodJob::Execution` but referring to jobs. I think the language is an improvement.

**Pull request merged**